### PR TITLE
test(plugin-meetings): remove nested its

### PR DESCRIPTION
**Pull request by edvujic**
# COMPLETES [WEBEX-390386](https://jira-eng-gpk2.cisco.com/jira/browse/WEBEX-390386)

## This pull request addresses

We've encountered a problem within the unit tests for the stats analyzer where nested it statements are not executing as intended. Specifically, when an it block, which is meant to define a specific test case, is placed inside another it block, the inner it is being overlooked and not run.

The reason for this behavior is that the testing framework we are using is designed to support a flat structure of describe and it blocks. 

## by making the following changes
```javascript
it('...', () => {
  it('...', () => {});
  it('...', () => {});
});
```

to
```javascript
describe('...', () => {
  it('...', () => {});
  it('...', () => {});
});

```
- changed some test names for more clarity

### Change Type

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [X] Internal code refactor

### I certified that

- [X] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [X] I discussed changes with code owners prior to submitting this pull request

- [X] I have not skipped any automated checks
- [X] All existing and new tests passed
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js [hunk 1]
```diff
@@ -1673,7 +1689,7 @@ describe('plugin-meetings', () => {
                     type: 'outbound-rtp',
                     bytesSent: 1000,
                     framesSent: 1,
-                    packetsSent: 1,
+                    packetsSent: 0,
```
Comment (Shreyas281299) [inline packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js:1692]: Query: why wasn't this caught before?

**Review by Shreyas281299 — APPROVED** [review]
Changes look good. Thanks for identifying and fixing this. This is a learning for me as well. Just a non blocking comment.

**Comment by Shreyas281299** [comment]
@edvujic is statsAnalyser the only place where we were doing this? Or are there more plugins where this pattern was followed? I think we would find more places where this was followed. 

@mkesavan13

**Review by brycetham — COMMENTED** [review]
Looks good but I am also curious about @Shreyas281299's question.

File: packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js (hunk 1 above)
Comment (brycetham) [reply packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js:1692]: @edvujic could you double check to see why this change was needed?

File: packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js [hunk 2]
```diff
@@ -1646,189 +1662,191 @@ describe('plugin-meetings', () => {
...
-          screenShareVideo: {
-            senders: [fakeStats.video.senders[0]],
-            receivers: [fakeStats.video.receivers[0]],
-          },
-        });
-
-        await startStatsAnalyzer({expected: {receiveVideo: true}});
-
-        await progressTime();
+      describe('stream count for simulcats', async() => {
```
Comment (brycetham) [inline packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js:1665]: Typo: "simulcast"

**Review by k-wasniowski — APPROVED** [review]

**Pull request merged**